### DISCUSSION TITLE
request api, preview show had data, but table cannot showed

### DISCUSSION THREAD

**Discussion by Pierrelai99** [Budibase/budibase#5210] [Help]
hi budibase, I am trying to pass in 2 parameters: fagent and fpin, at preview there can show data, but not sure why the table didn't show the values , is that had bugs? 
![image](https://user-images.githubusercontent.com/99549872/161266183-9f362467-6a2d-4a7f-b004-f1d675de64db.png)

![image](https://user-images.githubusercontent.com/99549872/161266047-5ac8ba65-2220-4ca5-85b9-997ef08dcd2b.png)
while when I only pass 1 parameters then can see the data , i think is bugs help check xia ,thanks

**Comment by melohagan** [comment]
Hi @Pierrelai99 

Would you mind sharing the setup of your REST query please? What does your transformer look like for example.
